### PR TITLE
refactor: organize generate tests to improve CI times

### PR DESCRIPTION
**Pull request by Brocco**

### CODE REVIEW

File: tests/e2e/tests/generate/generate-util.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+export interface TestGenerateParam {
+  blueprint: 'component' | 'directive' | 'guard' |
+    'module' | 'pipe' | 'service';
+  name: string;
+  flags: string[];
+  pathsToVerify: string[];
+}
+
+export function testGenerate(params: TestGenerateParam): Promise<any> {
+  return ng('generate', params.blueprint, params.name, ...params.flags)
```
Comment (hansl) [inline tests/e2e/tests/generate/generate-util.ts]: Could you do `const { blueprint, name, flags, pathsToVerify } = params`? It would simplify the code.

Comment (Brocco) [inline tests/e2e/tests/generate/generate-util.ts]: Sure thing, good refactor/reorganization idea.

**Comment by Brocco** [comment]
@hansl your concern has been addressed and this PR is ready to be reviewed

File: tests/e2e/utils/generate.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    'module' | 'pipe' | 'service';
+  name: string;
+  flags?: string[];
+  pathsToVerify?: string[];
+}
+
+export function testGenerate(
+  {blueprint, name, flags, pathsToVerify}: TestGenerateParam): Promise<any> {
+
+    flags = flags || [];
```
Comment (hansl) [inline tests/e2e/utils/generate.ts]: Indentation is wrong. These two lines should be indented by 2, and the arguments should be by 4.

**Comment by filipesilva** [comment]
We should still run the unit tests though, and they seem to be absent from `testGenerate`. To reduce the number we can unit test at the end of a bunch of these... like all the component generators + unit test, etc.

**Pull request closed without merging**